examples/java_postgresql_customvar/README.md

```diff
@@ -112,48 +68,48 @@ This makes the `Database` custom resource available, that the application develo
 Annotations are created to describe the values that should be collected and made available by the Service Binding's intermediary secret.
 
 
-```
+```shell
 oc get crd databases.postgresql.baiju.dev -o yaml
 ```
 
 The database CRD contains the list of annotations:
 
-```
+```yaml
 metadata:
   annotations:
-    service.binding/dbName: 'path={.spec.dbName}'
-    service.binding/db.host: 'path={.status.dbConfigMap},objectType=ConfigMap'
-    service.binding/db.name: 'path={.status.dbConfigMap},objectType=ConfigMap'
-    service.binding/db.password: 'path={.status.dbConfigMap},objectType=ConfigMap'
-    service.binding/db.port: 'path={.status.dbConfigMap},objectType=ConfigMap'
-    service.binding/db.user: 'path={.status.dbConfigMap},objectType=ConfigMap'
-    service.binding/dbConnectionIP: 'path={.status.dbConnectionIP}'
-    service.binding/dbConnectionPort: 'path={.status.dbConnectionPort}'
-    service.binding/user: 'path={.status.dbCredentials},objectType=Secret'
-    service.binding/password: 'path={.status.dbCredentials},objectType=Secret'
-
+    service.binding/db.host: path={.status.dbConfigMap},objectType=ConfigMap,sourceValue=db.host
```

Comment:
I think sourceValue is redundant here.

Reply:
This is a list of annotations that comes from the DB operator's CRD out-of-the-box... this list just shows what is there after the DB operator is installed. So to change it (if needed), https://github.com/operator-backing-service-samples/postgresql-operator needs to be updated.

Reply:
After doublechecking, you are correct... the `sourceValue` is indeed not present out-of-the-box. thx